### ISSUE TITLE
external_pipeline needs source to exist before middleman starts

### ISSUE THREAD

**Issue by vvo** [middleman/middleman#1857]
I am using this config file options:

```rb
configure :development do
  activate :livereload
  activate :external_pipeline,
    name: :webpack,
    command: "webpack build into .webpack/javascripts/file.js",
    source: ".webpack"
end
```

With only this, when running latest middleman and middleman-livereload, it does not work if in my layout I add `<%= javascript_include_tag :file  %>` => middleman thinks the file does not exists (while its here on disk at the right localtion)

To solve this, I first need to have a first existing build in .webpack/javascripts/ then it all work, once the webpack build is rebuilt then livereload knows it and it works flawlessly.

So basically to fix that I did:

```sh
build .webpack/javascripts/file.js
middleman (and this will load the external pipeline command)
```

But I am wondering if this is expected behavior or a bug?

**Comment by vvo** [comment]
Seems like the only need is that the .webpack/javascripts/ folder exists. I was cleaning it everytime.

**Comment by tdreyno** [comment]
Hmm, that is a good question. I'm not sure MM should go around arbitrarily making folders for you (source could be anywhere on disk), but our file watcher does require an existing path to work.

Maybe, for now, add this to the top of your config:

```
require 'fileutils'
FileUtils.mkdir_p('.webpack')
```

**Comment by vvo** [comment]
Ok did that, I believe its a bit tricky because the default setup about webpack will not work (at least not on first launch)

Should we update the docs to indicate it?

**Comment by tdreyno** [comment]
Nah, I think I will do it automatically in MM. It'll be the least surprising
